dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -329,7 +368,17 @@ public boolean processEntryCommands(List<String> args, String activeCommand) thr
                 }
                 break;
             default:
-                return false;
+                List<String> possibleCommands = new ArrayList<String>();
```

Comment:
Implements suggestion tool for `dockstore workflow INVALID_COMMAND` and `dockstore tool INVALID_COMMAND`